src/cli/src/export.rs

```diff
@@ -216,6 +246,11 @@ pub struct Export {
     s3_access_key: Option<String>,
     s3_secret_key: Option<String>,
     s3_region: Option<String>,
+    oss: bool,
```

Comment:
Should we use a field like scheme or object_storage_type? @fengjiachun

Reply:
It's fine for me now; maybe we can do this after adding support for more object storage types.